### ISSUE TITLE
Plugin with id 'com.google.gms.google-services' not found.

### ISSUE THREAD

**Issue by Whip** [EddyVerbruggen/nativescript-plugin-firebase#1882]
My issue is identical to #278 but that was 5 years ago so maybe its different. I did try the things mentioned in that thread but no dice. Here's my case. 

I have an app that was last updated last year. I decided to do an yearly maintenance update. I updated Android SDK, XCode, NS CLI, NS Core, theme and plugins. Then ran

```
ns build android
```
and got this error
```
Script '/Users/Whip/Documents/ProjectName/node_modules/@nativescript/firebase/platforms/android/include.gradle' line: 96
A problem occurred evaluating script.
Plugin with id 'com.google.gms.google-services' not found.

Command ./gradlew failed with exit code 1
```
Here are my dependencies
```
"dependencies": {
    "@nativescript-community/ui-collectionview": "^4.0.49",
    "@nativescript-community/ui-material-cardview": "^7.0.5",
    "@nativescript-community/ui-material-textfield": "^7.0.5",
    "@nativescript-community/ui-material-textview": "^7.0.5",
    "@nativescript/core": "^8.2.3",
    "@nativescript/firebase": "^11.1.3",
    "@nativescript/theme": "^3.0.2",
    "@nstudio/nativescript-checkbox": "^2.0.5",
    "@nstudio/nativescript-loading-indicator": "^4.1.2",
    "@triniwiz/nativescript-image-cache-it": "^7.0.14",
    "@triniwiz/nativescript-image-zoom": "^4.1.1",
    "@triniwiz/nativescript-toasty": "^4.1.3",
    "@triniwiz/nativescript-youtubeplayer": "^4.1.4",
    "nativescript-exit": "1.0.1",
    "nativescript-gif": "^5.0.0",
    "nativescript-material-icons": "^1.0.3",
    "nativescript-permissions": "^1.3.12",
    "nativescript-phone": "^3.0.3",
    "nativescript-ui-autocomplete": "^8.0.1",
    "nativescript-ui-sidedrawer": "^10.0.2"
  },
  "devDependencies": {
    "@nativescript/android": "~8.2.2",
    "@nativescript/ios": "~8.2.3",
    "@nativescript/webpack": "^5.0.5-rc.0"
  }
```

Can you help me with this?

**Comment by weareframework** [comment]
Same issue here

**Comment by jeanpaulattard** [comment]
I had that very same issue. I ended up downgrading to `@nativescript/core` to version `8.2.0` to continue working. Not sure what the problem with this version is

**Comment by Whip** [comment]
You think the problem is with core and not this plugin?

**Comment by jeanpaulattard** [comment]
I'm not sure to be quite honest. Didn't give it much investigation after finding the fix for it.

**Comment by Whip** [comment]
Its sad to see this plugin is not being maintained right now. There are over 500 open issues and last update was almost 2 years ago. @EddyVerbruggen can you confirm if you're still working on this?

**Comment by mika-ella** [comment]
hey @Whip are you able to find solution for this? I am also facing this issue earlier this May.

**Comment by Whip** [comment]
No. I believe the plugin needs to be updated. @EddyVerbruggen could tell. I hope he hasn't abandoned the project.

**Comment by gyansoft786** [comment]
Same issue here. After upgrading to {N} 8.2.2. It was working file with NS 8.0. Looks like this plugin no longer maintained. @EddyVerbruggen would you please confirm if this plugin is no longer in development.

**Comment by gyansoft786** [comment]
looks like things has been changed. We are using outdated plugin.
Have a look at this https://github.com/NativeScript/firebase
It works for me fine.

**Comment by jwrascoe** [comment]
update your app.gradle with this:

apply plugin: 'com.google.gms.google-services' 

Put it at the top of the file and should be fine.

**Comment by ishiharas** [comment]
The same problem is occuring to me after the migration from **nativescript 8.1 to 8.2.**

@jwrascoe 's fix worked for me (with a little modification), which is a nice "workaround". But on the long run the plugin should probably be declared as deprecated (@EddyVerbruggen ) and replaced by https://github.com/NativeScript/firebase .

```
repositories {
  google()
  mavenCentral()
}
dependencies {
  classpath 'com.google.gms:google-services:4.3.13'
  classpath 'com.google.firebase:firebase-crashlytics-gradle:2.9.1'
}
```

**Comment by machadoastur** [comment]
jwrascoe 
Could you tell me where to put that file? I can't find that code to include firebase in an Android Studio Firebase project

**Comment by jwrascoe** [comment]
@machadoastur It goes in your App_Resources/Android directory where your Angular code is, not in the platform generated code... the NS builder looks at that file and then creates the correct files in the platform area

**Comment by machadoastur** [comment]
Muchas gracias.
